connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -143,7 +143,7 @@ export async function githubUpsertIssueActivity(
   const documentId = getIssueDocumentId(repoId.toString(), issueNumber);
   const issueAuthor = renderGithubUser(issue.creator);
   const tags = [
-    `title:${issue.title}`,
+    `title:${issue.title.substring(0, 512)}`,
```

Comment:
can we do it [here](https://github.com/dust-tt/dust/blob/8bd2c8c8e2b7444f54bf5c182aafb7f04ee28f3e/connectors/src/lib/data_sources.ts#L103) instead ?

Reply:
We IRL discussed it to do directly in the connectors to only do it for the title but I'll make the change 👍🏻 
Adding a commit!

Reply:
I didn't realize that we discussed that. IMO it's better to do it only in one place on the whole tags array since this length limitation applies to all tags VS just the title, right

Reply:
Yes agreed and added the commit!

Reply:
🙏